identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-username-request.jsp

```diff
@@ -1507,6 +1507,10 @@
             $("#alphanumericUsernameField").show();
         }
 
+        if (<%=isEmailUsernameEnabled%> && <%=hideUsernameFieldWhenEmailAsUsernameIsEnabled%>) {
```

Comment:
`hideUsernameFieldWhenEmailAsUsernameIsEnabled` is this a new configuration or already existing one? If this is a new one can't we simply say `hideUsernameField` without coupling it with email as username feature?